lib/matplotlib/backends/backend_qt4.py

```diff
@@ -28,22 +28,14 @@
 
 class FigureCanvasQT(FigureCanvasQT5):
 
-    def __init__(self, figure):
-        if DEBUG:
-            print('FigureCanvasQt qt4: ', figure)
-        _create_qApp()
-
+    def _fake_super_fcq(self, figure):
         # Note different super-calling style to backend_qt5
         QtWidgets.QWidget.__init__(self)
-        FigureCanvasBase.__init__(self, figure)
-        self.figure = figure
```

Comment:
Because that stuff is handled in the `FigureCanvasQt5` init and this function injected in.